src/lib/stitching/gravity/stitching.ts

```diff
@@ -74,6 +81,35 @@ export const gravityStitchingEnvironment = (
             })
           },
         },
+        formattedEndAt: {
+          fragment: gql`
+            ... on ViewingRoom {
+              startAt
+              endAt
+              timeZone
+            }
+          `,
+          resolve: ({ startAt: _startAt, endAt: _endAt, timeZone }) => {
+            const startAt = moment(_startAt)
+            const endAt = moment(_endAt)
+
+            timeZone = timeZone || moment.tz.guess()
+
+            if (endAt > moment().add(30, "days")) {
+              return null
+            }
+            if (endAt < startAt) {
+              return "Closed"
+            }
+
+            const label =
+              "Closes in " +
+              (moment.duration(
+                endAt.tz(timeZone).diff(startAt)
```

Comment:
Perhaps this should be relative to now rather than `startAt`?

Reply:
@starsirius - would you mind pasting a code snippet of what you'd like to see here?